code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -211,9 +211,7 @@
 		M.adjustBruteLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
 		M.adjustFireLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
 	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(!H.species.exotic_blood && !(H.species.flags & NO_BLOOD) && prob(33))
-			H.vessel.add_reagent("blood", 1)
+		M.blood_volume += 1
```

Comment:
There needs to be a 33% probability added to this, otherwise this will be insanely powerful.